independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/util/RuntimeResourceVisitor.java

```diff
@@ -19,6 +19,9 @@ public default void visitRuntimeResource(String httpMethod, String fullPath, Run
     public default void visitEnd() {
     }
 
+    public default void visitStart() {
+    }
+
     public static void visitRuntimeResources(
             List<RequestMapper.RequestPath<RestInitialHandler.InitialMatch>> classMappers,
             RuntimeResourceVisitor visitor) {
```

Comment:
You have to call `visitStart` at the beginning of this method, otherwise it will not get called :)

Reply:
Okay :)